.github/workflows/pull-request.yml

```diff
@@ -21,3 +21,22 @@ jobs:
     uses: entur/gh-workflows/.github/workflows/pr-tests-helm.yml@main
     with:
       charts: '["common"]'
+
+  check-doc-gen:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Run helm-docs
+        uses: addnab/docker-run-action@v3
+        with:
+          image: jnorwood/helm-docs:latest
```

Comment:
Should set this to a major docker tag, or pin the version for renovate to handle